File: 백준/Silver/11399. ATM/ATM.py
n = int(input())
a = []
a = list(map(int, input().split(' ')))

a.sort()
sum = 0
for i in range(len(a)):
    sum += a[i]*(n-i)
# a[0]*(n) + a[1]*(n-1)

print(sum)

File: 백준/Silver/11399. ATM/ATM.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int N = Integer.parseInt(br.readLine());
		StringTokenizer st = new StringTokenizer(br.readLine());
		int[] atm = new int[N];
		for (int i = 0; i < N; i++) {
			atm[i] = Integer.parseInt(st.nextToken());
		}
		
		Arrays.sort(atm);
		int sum = 0;
		int tmp = 0;
		for (int time : atm) {
			tmp += time;
			sum += tmp;
		}
		System.out.println(sum);
	}
}
